### ISSUE TITLE
image has stopped rendering

### ISSUE THREAD

**Issue by kosn1** [dompdf/dompdf#3086]
I am using DOMPDF to create a pdf document with a logo on it. Everything worked fine so far but since yesterday the logo is not rendering anymore. This is the code I use to create the pdf
```
$options = new Options();
$options->setIsRemoteEnabled(true);
$dompdf = new Dompdf($options);
$dompdf->loadHtml($contents);

// (Optional) Setup the paper size and orientation
$dompdf->setPaper('A4', 'portrait');

// Render the HTML as PDF
$dompdf->render();

// Output the generated PDF
$output = $dompdf->output();

//write and save the data
file_put_contents($file_name, $output);
```

inside my rendered HTML I display the img like this
```
<div class="header-wrapper">
        <img src="<?php echo get_stylesheet_directory_uri() . '/assets/images/logo.png' ?>" alt="logo">
 </div>
```

I do not understand why suddenly has stopped working. Now I get an error msg **Image not found or type unknown**
![image](https://user-images.githubusercontent.com/47416784/208715798-7f438a12-5469-49b5-bf05-afd66fafe003.png)

PS. I have triple-checked that the file exists in the specified directory

**Comment by bsweeney** [comment]
No system updates or anything like that? You might try [enabling extra logging](https://github.com/dompdf/dompdf/wiki/Debugging-Dompdf) to try and help find why the image is failing to render.

**Comment by kosn1** [comment]
I am not able to setup your configuration about extra logging. Could you provide more details how to adjust it on my code?

**Comment by bsweeney** [comment]
You are not able to do so why? That's all just basic PHP, so using your sample code:

```
ini_set("display_errors", true);
ini_set("error_log", "phperr.log");
ini_set("log_errors", true);
error_reporting(E_ALL);

$_dompdf_show_warnings = true; // record warnings generated by Dompdf
$_dompdf_debug = false; // output frame details for every frame in the document
$_DOMPDF_DEBUG_TYPES = [
    'page-break' => false // record information about page break determination
];

$options = new Options([
    "isRemoteEnabled => true,

    /* debugging options */
    "logOutputFile" => "log.html",
    "debugPng" => false, // extra messaging
    "debugKeepTemp" => false, // don't delete temp files
    'debugCss' => false, // output Style parsing information and frame details for every frame in the document
    'debugLayout' => false, // draw boxes around frames
    'debugLayoutLines' => false, // line boxes
    'debugLayoutBlocks' => false, // block frames
    'debugLayoutInline' => false, // inline frames
    'debugLayoutPaddingBox' => false // padding box
]);
$dompdf = new Dompdf($options);
$dompdf->loadHtml($contents);

// (Optional) Setup the paper size and orientation
$dompdf->setPaper('A4', 'portrait');

// Render the HTML as PDF
$dompdf->render();

// Output the generated PDF
$output = $dompdf->output(['compress' => 0]);

//write and save the data
file_put_contents($file_name, $output);
```

If you can't change the error reporting options leave those out. You can still set the runtime options for Dompdf.

**Comment by kosn1** [comment]
Alright I configured the error logging but there were no errors related with the image. I also tried to hardcode the logo's url/path to the img src but still nothing changed. The only workaround I found was to encode the image to base64 and then append its data to img src. It's just strange because it was working fine and suddenly the image stopped loading even though no changes were made.

**Comment by bsweeney** [comment]
I can't provide much more guidance, then. A base64 image doesn't change the parsing of the image just how it's retrieved. Dompdf wouldn't just stop working when it was working fine previously if there were no system changes.

You might try using a JPG image instead to see if it's related to an underlying library. PNG images generally undergo much more internal processing.

Is this a system you maintain or are you using a webhost? Could something in the underlying system have changed? Maybe one of the temporary directories is no longer accessible or full (though I would expect to see an error related to directory access)?

**Comment by kosn1** [comment]
I have tried with JPG but no luck either. I have also tested to a completely new file, to create a pdf with just an image and I was not able to load it there as well. It just refuses to render the image.

I am using a webhost and yes as you said there were no errors related to directory access when I enabled error logging.
Anyway I will have to use the base64 workaround I guess. 
Thank you for the tips and your help!
